### ISSUE TITLE
what dataset you used and how to run this code

### ISSUE THREAD

**Issue by YichiHuang** [gautamb85/Code_#1]
Hi,
Can you tell me what dataset you used and how to run this code?
Thanks.

**Comment by gautamb85** [comment]
Hi,

Sorry it took me so long to reply. 
I put that code on github for my own reference, it was not a package release. 

There is a bug in the code so it wont run. Not to mention I use a private dataset (belongs to my lab). 

It is not very hard to build a RNN or CNN yourself using keras, theano, tensorflow or lasagne (like me). Depends if u want to classify every frame or use one label for full recording (depends how long recordings are, how big your gpu is etc) 
You can use my code as reference and I am happy to help via email. 

Hope this helps,
Good luck!
Gautam
